storybook/src/pages/amsterdam/NavigationPage/NavigationPage.stories.tsx

```diff
@@ -0,0 +1,277 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import {
+  Card,
+  Grid,
+  Heading,
+  Image,
+  Link,
+  LinkList,
+  PageHeading,
+  Paragraph,
+  SearchField,
+  Spotlight,
+  UnorderedList,
+} from '@amsterdam/design-system-react'
+import {
+  CameraIcon,
+  ChatBubbleIcon,
+  FacebookIcon,
+  LinkedinIcon,
+  TwitterIcon,
+} from '@amsterdam/design-system-react-icons'
+import { Meta, StoryObj } from '@storybook/react'
+import { NavigationColumn } from './NavigationColumn'
+import { NavigationPage } from './NavigationPage'
+import { burgerzakenLinks, parkerenLinks } from './links'
+import { exampleHeading, exampleParagraph } from '../../../components/shared/exampleContent'
+import { commonMeta } from '../common/config'
+
+const meta = {
+  ...commonMeta,
+  title: 'Pages/Amsterdam.nl/Navigation Page',
+  component: NavigationPage,
+} satisfies Meta<typeof NavigationPage>
+
+export default meta
+
+export const Default: StoryObj = {
+  args: {
+    children: (
+      <Grid gapVertical="large" paddingBottom="large">
+        <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+          <PageHeading className="ams-mb--md">Burgerzaken</PageHeading>
+          <Paragraph size="large">
+            Heeft u een paspoort, rijbewijs of uittreksel nodig? Of wilt u een verhuizing doorgeven of een geboorte
+            aangeven? Op deze pagina vindt u alle informatie en regelzaken rondom Burgerzaken.
+          </Paragraph>
+        </Grid.Cell>
+        <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+          <NavigationColumn sections={burgerzakenLinks[0]} />
```

Comment:
I kind of expected the tab order to go down a list and then jump to the right-hand list on wider windows, so a zig zag pattern. That would mean there would be space below a short list next to a longer list, but maybe that would make the page a little less busy. Currently, the 6th most important list _(Rijbewijs)_ is next to the most important one, _(Paspoort en identiteitskaart)_. That's a little weird.

Reply:
Yes, we discussed this on Friday. Using a Grid Cell for each list does make everything simpler. Updated.